web_src/js/features/common-global.js

```diff
@@ -46,10 +46,11 @@ export function initFootLanguageMenu() {
 }
 
 export function initGlobalEnterQuickSubmit() {
-  $(document).on('keydown', '.js-quick-submit', (e) => {
-    if (((e.ctrlKey && !e.altKey) || e.metaKey) && (e.key === 'Enter')) {
+  document.addEventListener('keydown', (e) => {
+    const ieQuickSubmitEnter = ((e.ctrlKey && !e.altKey) || e.metaKey) && (e.key === 'Enter');
+    if (ieQuickSubmitEnter && e.target.matches('textarea')) {
```

Comment:
1. Why `ie`? What does Internet Explorer have to do with it?
2. I wonder if there will be any regression where a textarea was **explicitly** not supposed to submit.
But overall I prefer auto-submission for any textarea.

Reply:
Ooooooops .... typo ......... will fix soon

I think there should be no regression. No user would press Ctrl+Enter for any other reason at the moment.

Reply:
```suggestion
    const isQuickSubmitEnter = ((e.ctrlKey && !e.altKey) || e.metaKey) && (e.key === 'Enter');
    if (isQuickSubmitEnter && e.target.matches('textarea')) {
```